perceptia-one/perceptia/src/components/QuizInfo.js

```diff
@@ -41,11 +44,12 @@ class QuizInfo extends React.Component {
   }
 
   componentDidMount() {
-      fetch('http://students.washington.edu/long27km/any_quiz/v1/read/' + this.state.quiz)
+      fetch(constants.localhost.url + 'read/' + this.state.quiz)
```

Comment:
See comment in constants.js about using the api.url constant.

Reply:
Fixed